api/python/provisioner/utils.py

```diff
@@ -408,7 +408,7 @@ def generate_random_secret():
     Generates a random 14 character password
     """
 
-    special_characters = '=+-_'
+    special_characters = '=+-()_'
```

Comment:
Need to remove the "()", I believe must have got added through conflict resolution